spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/kafka/KafkaProperties.java

```diff
@@ -643,6 +663,81 @@ public void setRetries(Integer retries) {
 
 	}
 
+	public static class Admin {
+
+		private final Ssl ssl = new Ssl();
+
+		/**
+		 * Id to pass to the server when making requests; used for server-side logging.
+		 */
+		private String clientId;
+
+		/**
+		 * Additional producer-specific properties used to configure the client.
+		 */
+		private final Map<String, String> properties = new HashMap<>();
```

Comment:
I am confused by the number of `properties` object in this class. Is the Javadoc accurate?

Reply:
Typo in javadoc - should say `admin-specific`. Kafka has many, many, properties. You may recall that my initial PR (for kafka AutoConfig) had many more properties and I was told it was too much; I therefore selected the important ones, but provided the properties object to allow configuration of less-common properties.

Some properties are global; some apply to consumers, producers, and admins; again, I selected important ones, while allowing configuration of less-common properties for each entity.